Examples/Physics_applications/laser_acceleration/inputs_1d

```diff
@@ -45,6 +45,10 @@ electrons.profile = constant
 electrons.density = 2.e23  # number of electrons per m^3
 electrons.momentum_distribution_type = "at_rest"
 electrons.do_continuous_injection = 1
+electrons.addRealAttributes = orig_x
+electrons.attribute.orig_x(x,y,z,ux,uy,uz,t) = "x"
+electrons.addIntegerAttributes = upstream
+electrons.attribute.upstream(x,y,z,ux,uy,uz,y) = " (x<0)*1"
```

Comment:
`x` is always zero in 1D and `z` is also a bit more useful for acceleration/injection here:
```suggestion
electrons.addRealAttributes = orig_z
electrons.attribute.orig_z(x,y,z,ux,uy,uz,t) = "z"
electrons.addIntegerAttributes = upstream
electrons.attribute.upstream(x,y,z,ux,uy,uz,y) = "(z<0)"
```